src/data_info/clients/notifications.clj

```diff
@@ -30,8 +30,8 @@
 (defn move-notification
   [user start-paths end-paths failed?]
   (amend-notification
-    {:subject (str (completed-text failed?) " moving " (count start-paths) " file(s)/folder(s) to " (first end-paths))
-     :message (str (completed-text failed?) " moving " (string/join ", " start-paths) " to " (first end-paths))} 
+    {:subject (str (completed-text failed?) " moving " (count start-paths) " file(s)/folder(s) to " (ft/dirname (first end-paths)))
+     :message (str (completed-text failed?) " moving " (string/join ", " start-paths) " to " (string/join ", " end-paths))}
```

Comment:
These two changes are to handle move notifications now passing a list of the paths moved-to.